### ISSUE TITLE
Placeholder not completely replaced

### ISSUE THREAD

**Issue by itsamaik** [antonmihaylov/OpenXmlTemplates#7]
If you set a placeholder in your control with a text length longer than the value that is going to replace it, it will only replace the length of the given value. For example, if the placeholder text is "abcdefgh" and you replace it with "Test", you will get "Testefgh"

**Comment by antonmihaylov** [comment]
I couldn't replicate that, can you give me an example (doc file and data file)

**Comment by itsamaik** [comment]
I've managed to replicate it with this document: [debug.docx](https://github.com/antonmihaylov/OpenXmlTemplates/files/4821689/debug.docx)

I've used the following data:
`new
{
      test = "abc",
      show = true,
}`

Resulting in this document: [debug_output.docx](https://github.com/antonmihaylov/OpenXmlTemplates/files/4821697/debug_output.docx)

I am still not completely sure when it happens. When I cleared out my document and left one control it worked. However, then I created a completely new document and added the control + a table and some controls and it resulted into this. When I remove the table, the issue still occurs, so I don't think the table has any influence.
